src/runtime/virtcontainers/fs_share_linux.go

```diff
@@ -456,6 +527,10 @@ func (f *FilesystemShare) ShareRootFilesystem(ctx context.Context, c *Container)
 		}, nil
 	}
 
+	if HasOptionPrefix(c.rootFs.Options, VirtualVolumePrefix) {
```

Comment:
This serves as an initial step towards extending `KataVirtualVolume` to support data volumes. Presently, our support is limited to rootfs(`containerStorages`).

Reply:
Not a must do, but I wonder if it's worth highlighting that in the commit message as from reading the current version I was expecting to see code for that in the commit, rather than it being future prep?

Reply:
Sure. I'll mention that in the commit. Thanks for you suggestions.